### ISSUE TITLE
Not compiling with icu 61.1

### ISSUE THREAD

**Issue by quite** [arybczak/ncmpcpp#285]
ncmpcpp seems to have been hurt by icu going 60.2->61.1 on this Arch system

```
./regex_filter.h:45:22: error: ‘UnicodeString’ has not been declared
  static void convert(UnicodeString &s)
                      ^~~~~~~~~~~~~
./regex_filter.h:61:9: error: ‘Transliterator’ does not name a type; did you mean ‘UTransliterator’?
  static Transliterator *m_converter;
         ^~~~~~~~~~~~~~
```

**Comment by arybczak** [comment]
Does https://github.com/arybczak/ncmpcpp/commit/7a75756ca30f9627f6e3cfca081b108f1976d4a4 help?

**Comment by quite** [comment]
Ah, had the feeling it had to do with scoping. Let `StringPiece` be scoped too, and we're good!

**Comment by arybczak** [comment]
Alright, fixed by e3e35dc, thanks. I'm going to release 0.8.2 shortly.

**Comment by quite** [comment]
Thank You!
